R/library.R

```diff
@@ -81,6 +81,13 @@ install_package.local <- function(locked_package, libPath, quiet) {
     , type = "source", quiet = quiet)
 }
 
+install_package.tarball <- function(locked_package, libPath, quiet) {
```

Comment:
Does it make sense to use the tarball that was downloaded when checking dependencies, in the same spirit as what we do with [CRAN packages](https://github.com/robertzk/lockbox/blob/master/R/library.R#L85-L89)?

Reply:
@kirillseva Can you address?

Reply:
will do, good catch! 👀